pig/accumulo.pig/src/main/java/mvm/rya/accumulo/pig/StatementPatternStorage.java

```diff
@@ -289,11 +289,14 @@ public Tuple getNext() throws IOException {
                 RyaStatement ryaStatement = ryaContext.deserializeTriple(layout, new TripleRow(key.getRow().getBytes(),
                         key.getColumnFamily().getBytes(), key.getColumnQualifier().getBytes()));
 
-                Tuple tuple = TupleFactory.getInstance().newTuple(4);
+                Tuple tuple = TupleFactory.getInstance().newTuple(7);
                 tuple.set(0, ryaStatement.getSubject().getData());
                 tuple.set(1, ryaStatement.getPredicate().getData());
                 tuple.set(2, ryaStatement.getObject().getData());
                 tuple.set(3, (ryaStatement.getContext() != null) ? (ryaStatement.getContext().getData()) : (null));
+                tuple.set(4, ryaStatement.getSubject().getDataType());
+                tuple.set(5, ryaStatement.getPredicate().getDataType());
+                tuple.set(6, ryaStatement.getObject().getDataType());
```

Comment:
Not a big deal, but you only need to set the data type on the object -- subjects/predicates are always uris by default.  not a huge issue, I'll merge it regardless.